pkg/arvo/mar/publish/info.hoon

```diff
@@ -45,8 +45,8 @@
       ==
     ++  new-parser
       ;~  plug
```

Comment:
`;~ (glue gaq)` instead of having `sfix ... gaq` in `key-val`? Would let the "optional trailing newline" `pose`s be cleaned up to just `sfix ... (punt gaq)`
Alternately, you could define `++  eol  ;~(pose gaq (full (easy ~)))` for line-or-EOF termination